wasm/src/main/scala/ir2wasm/Preprocessor.scala

```diff
@@ -65,6 +65,7 @@ object Preprocessor {
         clazz.interfaces.map(_.name),
         clazz.ancestors,
         !clazz.hasDirectInstances,
+        clazz.hasRuntimeTypeInfo || clazz.hasInstanceTests,
```

Comment:
I'm wondering what kind of classes have `hasRuntimeTypeInfo` true?
And why we want to make it `true` when `clazz.hasInstanceTests` is true ?

Reply:
I elaborated in a comment. Is it better?

Reply:
Thanks! that explains a lot!